DOCS: fix fatal issue with es_ES locale

patrickabolger opened the pull request:
`tips.txt` was translated as `consejos.txt`, which would have caused a crash if someone had chosen this locale. I set it back to `tips.txt`. (if someone wants start-up tips in Spanish, they'll need a `tips_es_ES.txt` file for Spanish in Spain, or `tips_es_CO.txt` file for Spanish in Colombia) Also, the locale is `es_ES`, but someone changed the language within that locale to Spanish in Colombia (which should be a new locale: `es_CO`, instead). I changed the language in `es_ES` to Spanish in Spain (as it must be). I'm not sure if the translations in there are for Colombian or Iberian Spanish. I'll need to look closer. Or we can ask whoever made the pull request.

patrickabolger committed:
DOCS: fix fatal issue with es_ES locale

tips.txt was translated as consejos.txt, which would have caused a crash. The locale is es_ES, but someone changed the language to Spanish in Colombia (which should be a new locale: es_CO, instead)

shun2wang commented on psychopy/app/locale/es_ES/LC_MESSAGE/messages.po:
```diff
@@ -300,9 +300,11 @@ msgstr ""
+#, fuzzy
 msgid "tips.txt"
-msgstr "consejos.txt"
+msgstr "tips.txt"
```
we should also remove the call of  `_translate()` from origin translation strings at https://github.com/psychopy/psychopy/blob/11a53f5420330d6c4080c3e1219b46a3a7258f21/psychopy/app/_psychopyApp.py#L654, then this `msgid "tips.txt"` will not get into the .po files. that will be safety for this issue.

shun2wang replied:
and make file links for translation languages of tips.txt

patrickabolger committed:
DOCS: change reference language of es_ES back to Iberian Spanish

This was set to Colombian Spanish for some reason. es_ES is Iberian Spanish; es_CO is Colombian spanish.
